what4/src/What4/Expr/Builder.hs

```diff
@@ -2558,6 +2900,14 @@ instance IsExprBuilder (ExprBuilder t st fs) where
       Just LeqProof <- return $ isPosNat w
       bvUnary sym $ UnaryBV.uext u w
 
+    | Just (BaseIte _ _ c a b) <- asApp x
+    , Just a_bv <- asBV a
+    , Just b_bv <- asBV b = do
+      Just LeqProof <- return $ isPosNat w
+      a' <- bvLit sym w $ BV.zext w a_bv
+      b' <- bvLit sym w $ BV.zext w b_bv
+      bvIte sym c a' b'
+
```

Comment:
Am I correct in assuming all these general `BaseIte` are not related to LIA CHC and corresponding transformations and are just missing in our standard ExprBuilder logic?